determined-ai/determined
# fix: Run environment bump scripts after environments fix

## Description

Run environmental bump script after merging environment changes to the environment repo (commit hash: `835d8b14bd12b08bf3a19f29f5a3c23ccb1e40cb`).

Followed instructions here: https://github.com/determined-ai/determined/tree/master/tools/scripts

## Test Plan

### Tested on AWS
Used image corresponding to determined repo commit hash `a908a269854cc26537f92efff89d16b0ff2ef21d`
- Ensure that the environment image by default correspond with `835d8b14bd12b08bf3a19f29f5a3c23ccb1e40cb`
- Ensure that Jupyterlab starts normally
<img width="1141" alt="Screen Shot 2023-03-20 at 9 43 47 PM" src="https://user-images.githubusercontent.com/16422598/226678449-943b6936-0912-4425-a1f4-2f5b72201f3b.png">
<img width="2501" alt="Screen Shot 2023-03-20 at 9 43 34 PM" src="https://user-images.githubusercontent.com/16422598/226678484-32e75f6b-dcfb-4a60-8fb5-e47700dab5e0.png">


### Tested locally
Built cluster locally on commit hash `a908a269854cc26537f92efff89d16b0ff2ef21d`
- Ensure that the environment image by default correspond with `835d8b14bd12b08bf3a19f29f5a3c23ccb1e40cb`
- Ensure that Jupyterlab starts normally
<img width="2504" alt="Screen Shot 2023-03-21 at 9 32 30 AM" src="https://user-images.githubusercontent.com/16422598/226678560-16907d58-3c8f-41b1-893e-c8066c649bbf.png">
<img width="2475" alt="Screen Shot 2023-03-21 at 9 33 04 AM" src="https://user-images.githubusercontent.com/16422598/226678631-4a98b2fe-7ced-4431-80e5-741c2c31ca9f.png">


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
MLG-363
